main/FatesHistoryInterfaceMod.F90

```diff
@@ -6764,6 +6770,12 @@ subroutine define_history_vars(this, initialize_variables)
             upfreq=group_dyna_simple, ivar=ivar, initialize=initialize_variables,                &
             index = ih_npp_stor_si)
 
+       call this%set_history_var(vname='FATES_GRAZING', units='kg m-2 s-1',    &
+            long='grazing by herbivores of leaves in kg carbon per m2 per second',          &
```

Comment:
This is great to have this FATES_GRAZING output and know the mass of leaves eaten.  In the future could we also have a FATES_GRAZING_LITTER_OUT type output, that gives us an idea of the amount of manure output. Or helps to differentiate how much litter is natural from turnover vs. litter from herbivores.  And maybe useful later for CH4 tracking?